File: lesson4/c_c++/homework/homework4_33.c
//: lesson4/c_c++/homework/homework4_33.c
// *Ввести натуральное число N и вывести значение число 1/N, выделив период
// дроби. Например: 1/2=0.5 или 1/7=0.(142857).
#include <stdio.h>

int main() {
    int first_value = 1, second_value;
    float division_result;

    scanf("%d", &second_value);
    division_result = first_value / (float)second_value;

    if (second_value > 2 && second_value < 10) {
        printf("%d/%d=%f\n", first_value, second_value, division_result);
    } else {
        if (second_value == 10) {
            printf("%d/%d=%.1f\n", first_value, second_value, division_result);
        } else {
            printf("%d/%d=%.1f\n", first_value, second_value, division_result);
        }
    }
    return 0;

}

File: lesson4/python3/homework/homework4_33.py
#!/usr/bin/python3
#: lesson4/python3/homework/homework4_33.py

first_value = 1
second_value = int(input())
division_result = first_value / second_value
result = "%.6f" % division_result

if second_value > 2 and second_value < 10:
    print(str(first_value) + '/' + str(second_value) + '=' + str(result))
else:
    print(str(first_value) + '/' + str(second_value) + '=' + str(division_result))

